yb-voyager/src/migassessment/assessmentDB.go

```diff
@@ -214,3 +227,34 @@ func (adb *AssessmentDB) PopulateMigrationAssessmentStats() error {
 
 	return nil
 }
+
+func (adb *AssessmentDB) FetchAllStats() (*[]MigrationAssessmentStats, error) {
+	log.Infof("fetching all stats info from %q table", MIGRATION_ASSESSMENT_STATS)
+	query := fmt.Sprintf(`SELECT schema_name, object_name, row_count, column_count, reads, writes, is_index, parent_table_name, size_in_bytes FROM %s;`, MIGRATION_ASSESSMENT_STATS)
+	rows, err := adb.db.Query(query)
+	if err != nil {
+		return nil, fmt.Errorf("error querying all stats-%s: %w", query, err)
+	}
+	defer rows.Close()
+
+	var stats []MigrationAssessmentStats
+	for rows.Next() {
+		var stat MigrationAssessmentStats
+		var rowCount, columnCount sql.NullInt64
+		var parentTableName sql.NullString
+		if err := rows.Scan(&stat.SchemaName, &stat.ObjectName, &rowCount, &columnCount, &stat.Reads, &stat.Writes, &stat.IsIndex, &parentTableName, &stat.SizeInBytes); err != nil {
+			return nil, fmt.Errorf("error scanning row: %w", err)
+		}
+
+		stat.RowCount = lo.Ternary(rowCount.Valid, rowCount.Int64, -1)
+		stat.ColumnCount = lo.Ternary(columnCount.Valid, columnCount.Int64, -1)
```

Comment:
Why set them to `-1`?

Reply:
-1 will denote NULL here, in database rowCOunt is NULL for object_type=Index.
But in json we need something, rowCount=0 can be misinformation for indexes i think?